gaiku-3d/src/bakers/marching_cubes/mod.rs

```diff
@@ -29,7 +27,14 @@ impl GridCell {
         let iso: Vec3 = [isolevel, isolevel, isolevel].into();
 
         if (point1 - point2).abs() > [0.00001, 0.00001, 0.00001].into() {
-            (point1 + (point2 - point1) / (point2 - point1) * (iso - point1)).into()
+            // #FIXME msg from QEAndy to Norman
+            // Possible issue here:
+            // Originally this said
+            // (point1 + (point2 - point1) / (point2 - point1) * (iso - point1)).into()
+            // But (point2 - point1) / (point2 - point1) = [1., 1., 1.]
+            // Is this intentional?
+            // Not sure so swapped it for an explicit [1., 1., 1.]
+            (point1 + Vec3::one() * (iso - point1)).into()
```

Comment:
P.s. Why was this `(point2 - point1) / (point2 - point1)` shouldn't that just be a unit vector. Maybe there is a missing bracket?

Reply:
To be honest, I don't know, when I wrote this baker I was porting an existing C implementation, but I ended up "tweaking" a bit so I might messed up and lastly I'm not 100% sure what this line does (or the entire function) :)

Checking at the results seems that this function was the one I changed and broke the baker, with your change the vertice of the triangle are pointing to the coord 0.0 from the current chunk, instead of the 0.0 of the entire scene, can you put this in a separate PR so I can merge the github actions fix?

Reply:
I can revert this part but it's one of the clippy issues (I was surprised to learn clippy can detect `x/x`) so I'll have to add a clippy allow directive here.

Reply:
What was your original c reference? Maybe I can have a poke at repairing it.

Reply:
Reverted and allow directive added instead

Reply:
> What was your original c reference?

I think so, sadly I don't remember which one anymore.